src/graphql-server/resolvers.js

```diff
@@ -99,9 +99,20 @@ export const resolvers = {
 
     holofuelAcceptOffer: (_, { transactionId }) => HoloFuelDnaInterface.offers.accept(transactionId),
 
-    holofuelDecline: (_, { transactionId }) => HoloFuelDnaInterface.transactions.decline(transactionId),
+    holofuelDecline: (_, { transactionId }) => {
+      console.log('!! INSIDE DECLINE ENDPOINT IN RESOLVERS !!', transactionId)
+      return HoloFuelDnaInterface.transactions.decline(transactionId)
+    },
+
+    holofuelCancel: (_, { transactionId }) => {
+      console.log('!! INSIDE CANCEL ENDPOINT IN RESOLVERS !!')
+      return HoloFuelDnaInterface.transactions.cancel(transactionId)
+    },
 
-    holofuelCancel: (_, { transactionId }) => HoloFuelDnaInterface.transactions.cancel(transactionId),
+    holofuelRecoverFunds: (_, { transactionId }) => {
+      console.log('!! INSIDE RECOVR FUNDS ENDPOINT IN RESOLVERS !!')
+      return HoloFuelDnaInterface.transactions.recoverFunds(transactionId)
+    },
```

Comment:
remove logs and make these one line definitions, like `holofuelAcceptOffer` above